CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
 - **General:** Use more readable timestamps in KEDA Operator logs ([#3066](https://github.com/kedacore/keda/issue/3066))
 - **Selenium Grid Scaler:** Edge active sessions not being properly counted ([#2709](https://github.com/kedacore/keda/issues/2709))
 - **Selenium Grid Scaler:** Max Sessions implementation issue ([#3061](https://github.com/kedacore/keda/issues/3061))
+- **Prometheus Scaler:** Add ignoreNullValues to return error when prometheus return null in values ([#3065](https://github.com/kedacore/keda/issues/3065))
```

Comment:
Please, move this line between **General:** and **Selenium Grid Scaler:** for sorting the changelog :pray:

Reply:
> The change looks good in general. Thanks for this improvement Some nits to be addressed:
> 
> * There is a test failing `TestPrometheusScalerCortexHeader`
> * Open a PR in [keda-docs](https://github.com/kedacore/keda-docs) updating the doc about prometheus scaler (only in the latest version)
> * Inline comment in changelog
> 
> Apart from that, IDK if it makes sense adding another e2e test to check this new behaviour, WDYT?

The changelog and `TestPrometheusScalerCortexHeader` already updated.
I will update the keda-doc these days.
I think it unnecessary to do e2e in this case, cause I already cover this one in test file by mocking the prometheus return data, which will be the same thing in e2e.

Reply:
updated